DateTimePickerLibrary/src/main/java/io/doist/datetimepicker/time/RadialTimePickerView.java

```diff
@@ -181,6 +182,28 @@
 
     private boolean mInputEnabled = true;
 
+    private static final int AMPM_INDEX = 2;
+    private static final int ENABLE_PICKER_INDEX =3;
```

Comment:
Can you comment on both of these with their original declarations, ie. `// TimePickerDialog.AMPM_INDEX` and so on? In case we add `TimePickerDialog` in the future.